### PR TITLE
Add top border back to inactive chapters

### PR DESCRIPTION
**Pull request by duarte**
This adds the top border back to inactive chapters, so the text doesn't jump down 1px when they become active.

### CODE REVIEW

File: src/css/_docs.scss [hunk 1]
```diff
@@ -435,6 +435,7 @@
     font-size: 15px;
     line-height: 17px;
     position: relative;
+    border-top: 1px solid #333;
```
Comment (akash1810) [inline src/css/_docs.scss]: Could the colour value be `transparent`? Should mean the `-active` colour can remain `$c-white` too.

Comment (duarte) [inline src/css/_docs.scss]: Well, the background here is #000, so the #333 is a soft grey line for inactive chapters

Comment (akash1810) [inline src/css/_docs.scss]: ah ok, I thought the inactive state didn't need a colour, but this looks fine

**Comment by akash1810** [comment]
👍

**Pull request merged**